sdk/servicebus/service-bus/samples/javascript/useProxy.js

```diff
@@ -36,14 +37,18 @@ async function main() {
       // No need to pass the `WebSocket` from "ws" package if you're in the browser
       // in which case the `window.WebSocket` is used by the library.
       webSocket: WebSocket,
-      webSocketConstructorOptions: { agent: proxyAgent }
-    }
+      webSocketConstructorOptions: { agent: proxyAgent },
+    },
```

Comment:
```suggestion
      webSocketConstructorOptions: { agent: proxyAgent }
    }
```
?

Reply:
Same thing - this is what prettier is applying.